python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -39,59 +40,98 @@ def transpile(self, program, place, scope=None):
             scope = global_scope()
         if not isinstance(scope, core.Scope):
             raise TypeError("scope should be as Scope type or None")
-        self.fuse_batch_norm(program, place, scope)
+        use_mkldnn = bool(os.getenv("FLAGS_use_mkldnn", False))
+        if not use_mkldnn:
+            self.fuse_batch_norm(program, place, scope)
+        else:
+            self.fuse_relu(program)
```

Comment:
- fuse_batch_norm is also suitable for MKLDNN. 
- After `fuse_batch_norm`, `fuse_relu` is conv+relu, do you mean that?

Reply:
1) If `fuse_batch_norm` is suitable for MKLDNN, then I'll leave it without checking `use_mkldnn` flag.
2) fuse_relu deletes relu from "batch norm + relu" pair.

Reply:
Since we have `fuse_batch_norm` already, there is no `batch_norm` op in inference program. Thus, what's the usage of `fuse_relu`?